_overviews/scala3-scaladoc/blog.md

```diff
@@ -33,4 +33,33 @@ Scaladoc loads blog if the `_blog` directory exists.
 All the blogpost filenames should start with date in numeric format matching `YYYY-MM-DD`.
 Example name is `2015-10-23-dotty-compiler-bootstraps.md`.
 
+## Page metadata
 
+The blog pages in scaladoc support [Yaml Frontmatter](https://assemble.io/docs/YAML-front-matter.html) which allows you to specify different values which will be used for metadata in your page. Here are the possible fields:
```

Comment:
this support is coming from Jekyll, I think? perhaps there's relevant Jekyll doc we could link to?